src/Validator/Interfaces/PhoneNumberValidatorInterface.php

```diff
@@ -0,0 +1,26 @@
+<?php
+declare(strict_types=1);
+
+namespace EoneoPay\Externals\Validator\Interfaces;
+
+interface PhoneNumberValidatorInterface
+{
+    /**
+     * Parses a phone number and returns a standardised formatted phone
+     * number.
+     *
+     * @param string $number
+     *
+     * @return string
+     */
+    public function format(string $number): ?string;
+
+    /**
+     * Checks if the phone number is valid.
```

Comment:
Valid means what? Valid in the context of the region? Valid international number?